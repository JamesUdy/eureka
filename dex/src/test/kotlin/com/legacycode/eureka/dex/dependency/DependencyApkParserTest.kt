package com.legacycode.eureka.dex.dependency

import com.legacycode.eureka.dex.test.TestApk
import com.legacycode.eureka.dex.test.TestGraphBuilder
import org.approvaltests.Approvals
import org.junit.jupiter.api.Test

class DependencyApkParserTest {
  @Test
  fun name() {
    // given
    val testApk = TestApk("wikipedia.apk")
    val parser = DependencyApkParser(testApk.file)

    // when
    val adjacencyList = parser.buildDependencyGraph()

    // then
    val dependencyGraph = adjacencyList.graph(TestGraphBuilder())
    Approvals.verify(dependencyGraph)
  }
}
