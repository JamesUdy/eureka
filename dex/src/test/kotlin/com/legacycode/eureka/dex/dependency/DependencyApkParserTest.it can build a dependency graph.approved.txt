Landroidx/fragment/app/Fragment; -> Landroidx/fragment/app/DialogFragment;
Landroidx/fragment/app/Fragment; -> Landroidx/preference/PreferenceFragmentCompat;
Landroidx/preference/EditTextPreferenceDialogFragmentCompat; -> Landroidx/preference/PreferenceFragmentCompat;
Landroidx/preference/ListPreferenceDialogFragmentCompat; -> Landroidx/preference/PreferenceFragmentCompat;
Landroidx/preference/MultiSelectListPreferenceDialogFragmentCompat; -> Landroidx/preference/PreferenceFragmentCompat;
Landroidx/fragment/app/DialogFragment; -> Landroidx/preference/PreferenceFragmentCompat;
Landroidx/fragment/app/Fragment; -> Lcom/bumptech/glide/manager/SupportRequestManagerFragment;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/gms/common/api/internal/zzd;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/material/datepicker/PickerFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/commons/FilePageFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/descriptions/DescriptionEditFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/descriptions/DescriptionEditSuccessFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/descriptions/DescriptionEditTutorialFragment$ItemFragment;
Lorg/wikipedia/descriptions/DescriptionEditTutorialFragment; -> Lorg/wikipedia/descriptions/DescriptionEditTutorialFragment$ItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/diff/ArticleEditDetailsFragment;
Lorg/wikipedia/page/linkpreview/LinkPreviewDialog; -> Lorg/wikipedia/diff/ArticleEditDetailsFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/diff/ArticleEditDetailsFragment;
Lorg/wikipedia/watchlist/WatchlistExpiryDialog; -> Lorg/wikipedia/diff/ArticleEditDetailsFragment;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/diff/ArticleEditDetailsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaAdvancedSettingsFragment;
Lorg/wikipedia/edit/insertmedia/InsertMediaImagePositionDialog; -> Lorg/wikipedia/edit/insertmedia/InsertMediaAdvancedSettingsFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaAdvancedSettingsFragment;
Lorg/wikipedia/edit/insertmedia/InsertMediaImageTypeDialog; -> Lorg/wikipedia/edit/insertmedia/InsertMediaAdvancedSettingsFragment;
Lorg/wikipedia/edit/insertmedia/InsertMediaImageSizeDialog; -> Lorg/wikipedia/edit/insertmedia/InsertMediaAdvancedSettingsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaSettingsFragment;
Lorg/wikipedia/views/ImagePreviewDialog; -> Lorg/wikipedia/edit/insertmedia/InsertMediaSettingsFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaSettingsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/preview/EditPreviewFragment;
Lorg/wikipedia/page/references/ReferenceDialog; -> Lorg/wikipedia/edit/preview/EditPreviewFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/edit/preview/EditPreviewFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/summaries/EditSummaryFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/feed/FeedFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/feed/configure/ConfigureFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/feed/news/NewsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/feed/onthisday/OnThisDayFragment;
Lorg/wikipedia/views/CustomDatePicker; -> Lorg/wikipedia/feed/onthisday/OnThisDayFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/feed/onthisday/OnThisDayFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/feed/suggestededits/SuggestedEditsCardItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/feed/topread/TopReadFragment;
Landroidx/fragment/app/DialogFragment; -> Landroidx/appcompat/app/AppCompatDialogFragment;
Landroidx/fragment/app/Fragment; -> Landroidx/appcompat/app/AppCompatDialogFragment;
Landroidx/fragment/app/DialogFragment; -> Landroidx/preference/PreferenceDialogFragmentCompat;
Landroidx/fragment/app/Fragment; -> Landroidx/preference/PreferenceDialogFragmentCompat;
Landroidx/fragment/app/DialogFragment; -> Lcom/google/android/gms/common/SupportErrorDialogFragment;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/gms/common/SupportErrorDialogFragment;
Lcom/google/android/material/datepicker/PickerFragment; -> Lcom/google/android/material/datepicker/MaterialCalendar;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/material/datepicker/MaterialCalendar;
Lcom/google/android/material/datepicker/MaterialDatePicker; -> Lcom/google/android/material/datepicker/MaterialCalendar;
Lcom/google/android/material/datepicker/MaterialCalendar; -> Lcom/google/android/material/datepicker/MaterialDatePicker;
Lcom/google/android/material/datepicker/PickerFragment; -> Lcom/google/android/material/datepicker/MaterialDatePicker;
Landroidx/fragment/app/DialogFragment; -> Lcom/google/android/material/datepicker/MaterialDatePicker;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/material/datepicker/MaterialDatePicker;
Lcom/google/android/material/datepicker/MaterialTextInputPicker; -> Lcom/google/android/material/datepicker/MaterialDatePicker;
Lcom/google/android/material/datepicker/PickerFragment; -> Lcom/google/android/material/datepicker/MaterialTextInputPicker;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/material/datepicker/MaterialTextInputPicker;
Lorg/wikipedia/onboarding/OnboardingFragment; -> Lorg/wikipedia/descriptions/DescriptionEditTutorialFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/descriptions/DescriptionEditTutorialFragment;
Landroidx/preference/PreferenceDialogFragmentCompat; -> Landroidx/preference/EditTextPreferenceDialogFragmentCompat;
Landroidx/fragment/app/Fragment; -> Landroidx/preference/EditTextPreferenceDialogFragmentCompat;
Landroidx/preference/PreferenceDialogFragmentCompat; -> Landroidx/preference/ListPreferenceDialogFragmentCompat;
Landroidx/fragment/app/Fragment; -> Landroidx/preference/ListPreferenceDialogFragmentCompat;
Landroidx/preference/PreferenceDialogFragmentCompat; -> Landroidx/preference/MultiSelectListPreferenceDialogFragmentCompat;
Landroidx/fragment/app/Fragment; -> Landroidx/preference/MultiSelectListPreferenceDialogFragmentCompat;
Landroidx/appcompat/app/AppCompatDialogFragment; -> Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment;
Landroidx/fragment/app/DialogFragment; -> Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment;
Landroidx/fragment/app/Fragment; -> Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/categories/CategoryDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/categories/CategoryDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/categories/CategoryDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImagePositionDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImagePositionDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImagePositionDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImageSizeDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImageSizeDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImageSizeDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImageTypeDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImageTypeDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/edit/insertmedia/InsertMediaImageTypeDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/gallery/GalleryItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/history/HistoryFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsTasksFragment; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/navtab/MenuNavTabDialog; -> Lorg/wikipedia/main/MainFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/feed/FeedFragment; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/history/HistoryFragment; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/readinglist/ReadingListsFragment; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/main/MainFragment;
Lorg/wikipedia/readinglist/MoveToReadingListDialog; -> Lorg/wikipedia/main/MainFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/onboarding/InitialOnboardingFragment$ItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/onboarding/OnboardingFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/page/PageFragment;
Lorg/wikipedia/page/references/ReferenceDialog; -> Lorg/wikipedia/page/PageFragment;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/page/PageFragment;
Lorg/wikipedia/watchlist/WatchlistExpiryDialog; -> Lorg/wikipedia/page/PageFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/page/PageFragment;
Lorg/wikipedia/theme/ThemeChooserDialog; -> Lorg/wikipedia/page/PageFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/page/customize/CustomizeToolbarFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/random/RandomFragment;
Lorg/wikipedia/random/RandomItemFragment; -> Lorg/wikipedia/random/RandomFragment;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/random/RandomFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/random/RandomFragment;
Lorg/wikipedia/readinglist/MoveToReadingListDialog; -> Lorg/wikipedia/random/RandomFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/random/RandomItemFragment;
Lorg/wikipedia/random/RandomFragment; -> Lorg/wikipedia/random/RandomItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/readinglist/ReadingListFragment;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/readinglist/ReadingListFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/readinglist/ReadingListFragment;
Lorg/wikipedia/readinglist/MoveToReadingListDialog; -> Lorg/wikipedia/readinglist/ReadingListFragment;
Lorg/wikipedia/readinglist/ReadingListsFragment; -> Lorg/wikipedia/readinglist/ReadingListFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/readinglist/ReadingListsFragment;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/readinglist/ReadingListsFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/readinglist/ReadingListsFragment;
Lorg/wikipedia/readinglist/MoveToReadingListDialog; -> Lorg/wikipedia/readinglist/ReadingListsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/search/RecentSearchesFragment;
Lorg/wikipedia/search/RecentSearchesFragment; -> Lorg/wikipedia/search/SearchFragment;
Lorg/wikipedia/search/SearchResultsFragment; -> Lorg/wikipedia/search/SearchFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/search/SearchFragment;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/search/SearchFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/search/SearchFragment;
Lorg/wikipedia/readinglist/MoveToReadingListDialog; -> Lorg/wikipedia/search/SearchFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/search/SearchResultsFragment;
Lorg/wikipedia/search/SearchFragment; -> Lorg/wikipedia/search/SearchResultsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/settings/languages/WikipediaLanguagesFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsCardsFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsCardsItemFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsCardsFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsItemFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsCardsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsItemFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsCardsFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsTasksFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/theme/ThemeFittingRoomFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/watchlist/WatchlistFragment;
Lorg/wikipedia/onboarding/OnboardingFragment; -> Lorg/wikipedia/onboarding/InitialOnboardingFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/onboarding/InitialOnboardingFragment;
Landroidx/preference/PreferenceFragmentCompat; -> Lorg/wikipedia/settings/PreferenceLoaderFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/settings/PreferenceLoaderFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsItemFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsCardsItemFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsCardsFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsCardsItemFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsCardsItemFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsImageTagDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsImageTagDialog;
Lorg/wikipedia/suggestededits/SuggestedEditsItemFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsImageTagsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsImageTagsFragment;
Lorg/wikipedia/suggestededits/SuggestedEditsImageTagDialog; -> Lorg/wikipedia/suggestededits/SuggestedEditsImageTagsFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/suggestededits/SuggestedEditsImageTagsFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/views/CustomDatePicker;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/views/CustomDatePicker;
Lorg/wikipedia/settings/PreferenceLoaderFragment; -> Lorg/wikipedia/settings/DeveloperSettingsFragment;
Landroidx/preference/PreferenceFragmentCompat; -> Lorg/wikipedia/settings/DeveloperSettingsFragment;
Lorg/wikipedia/settings/PreferenceLoaderFragment; -> Lorg/wikipedia/settings/NotificationSettingsFragment;
Landroidx/preference/PreferenceFragmentCompat; -> Lorg/wikipedia/settings/NotificationSettingsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/settings/NotificationSettingsFragment;
Lorg/wikipedia/settings/PreferenceLoaderFragment; -> Lorg/wikipedia/settings/SettingsFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/settings/SettingsFragment;
Landroidx/preference/PreferenceFragmentCompat; -> Lorg/wikipedia/settings/SettingsFragment;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/navtab/MenuNavTabDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/navtab/MenuNavTabDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/navtab/MenuNavTabDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/navtab/MenuNavTabDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/page/linkpreview/LinkPreviewDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/page/linkpreview/LinkPreviewDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/page/linkpreview/LinkPreviewDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/page/linkpreview/LinkPreviewDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/page/references/ReferenceDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/page/references/ReferenceDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/page/references/ReferenceDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/page/references/ReferenceDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/readinglist/AddToReadingListDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/readinglist/AddToReadingListDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/readinglist/AddToReadingListDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/readinglist/AddToReadingListDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/readinglist/ReadingListItemActionsDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/readinglist/ReadingListItemActionsDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/readinglist/ReadingListItemActionsDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/readinglist/SortReadingListsDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/readinglist/SortReadingListsDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/readinglist/SortReadingListsDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/theme/ThemeChooserDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/theme/ThemeChooserDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/theme/ThemeChooserDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/views/ImagePreviewDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/views/ImagePreviewDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/views/ImagePreviewDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/views/ImagePreviewDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/watchlist/WatchlistExpiryDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/watchlist/WatchlistExpiryDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/watchlist/WatchlistExpiryDialog;
Lorg/wikipedia/page/ExtendedBottomSheetDialogFragment; -> Lorg/wikipedia/wiktionary/WiktionaryDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/wiktionary/WiktionaryDialog;
Lcom/google/android/material/bottomsheet/BottomSheetDialogFragment; -> Lorg/wikipedia/wiktionary/WiktionaryDialog;
Landroidx/fragment/app/DialogFragment; -> Lorg/wikipedia/wiktionary/WiktionaryDialog;
Lorg/wikipedia/readinglist/AddToReadingListDialog; -> Lorg/wikipedia/readinglist/MoveToReadingListDialog;
Landroidx/fragment/app/Fragment; -> Lorg/wikipedia/readinglist/MoveToReadingListDialog;
